sql_grader/static/codemirror/lib/codemirror.js

```diff
@@ -0,0 +1,9693 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
This won't conflict with existing CodeMirror code in platform, right?

Reply:
TODO: Confirm Studio integration.